README.md

```diff
@@ -50,57 +57,35 @@ libraryDependencies += "com.google.cloud" % "google-cloud-bigquery" % "1.111.1"
---------------
+## Authentication
 
-See the [Authentication](https://github.com/googleapis/google-cloud-java#authentication) section in the base directory's README.
+See the [Authentication][authentication] section in the base directory's README.
 
-About Google Cloud BigQuery
---------------------------
+## Getting Started
 
-[Google Cloud BigQuery][cloud-bigquery] is a fully managed, NoOps, low cost data analytics service.
-Data can be streamed into BigQuery at millions of rows per second to enable real-time analysis.
-With BigQuery you can easily deploy Petabyte-scale Databases.
+### Prerequisites
 
-Be sure to activate the Google Cloud BigQuery API on the Developer's Console to use BigQuery from
-your project.
-
-See the [BigQuery client library docs][bigquery-client-lib-docs] to learn how to interact
-with Google Cloud BigQuery using this Client Library.
-
-Getting Started
----------------
-#### Prerequisites
-For this tutorial, you will need a
-[Google Developers Console](https://console.developers.google.com/) project with the BigQuery API
-enabled. You will need to [enable billing](https://support.google.com/cloud/answer/6158867?hl=en) to
-use Google Cloud BigQuery.
-[Follow these instructions](https://cloud.google.com/resource-manager/docs/creating-managing-projects) to get your
-project set up. You will also need to set up the local development environment by [installing the
-Google Cloud SDK](https://cloud.google.com/sdk/) and running the following commands in command line:
+You will need a [Google Cloud Platform Console][developer-console] project with the Cloud BigQuery [API enabled][enable-api].
+You will need to [enable billing][enable-billing] to use Google Cloud BigQuery.
+[Follow these instructions][create-project] to get your project set up. You will also need to set up the local development environment by
+[installing the Google Cloud SDK][cloud-sdk] and running the following commands in command line:
 `gcloud auth login` and `gcloud config set project [YOUR PROJECT ID]`.
 
-#### Installation and setup
+### Installation and setup
+
 You'll need to obtain the `google-cloud-bigquery` library.  See the [Quickstart](#quickstart) section
 to add `google-cloud-bigquery` as a dependency in your code.
 
-#### Creating an authorized service object
-To make authenticated requests to Google Cloud BigQuery, you must create a service object with
-credentials. You can then make API calls by calling methods on the BigQuery service object. The
-simplest way to authenticate is to use
-[Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
-These credentials are automatically inferred from your environment, so you only need the following
-code to create your service object:
+## About Cloud BigQuery
 
-```java
-import com.google.cloud.bigquery.BigQuery;
-import com.google.cloud.bigquery.BigQueryOptions;
 
-BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
```

Comment:
Note: this snippet didn't get moved to the custom content, but is may not actually be useful

Reply:
Yeah i think we can do without this snippet.